src/code/z_player_lib.c

```diff
@@ -1,153 +1,1021 @@
+        R_RUN_SPEED_LIMIT = 500;
+    }
+
+    if ((player->actor.id == ACTOR_PLAYER) && (player->transformation == PLAYER_FORM_FIERCE_DEITY)) {
+        scale = 0.015f;
+    } else {
+        scale = 0.01f;
+    }
+
+    Actor_SetScale(&player->actor, scale);
+}
+
+s32 Player_InBlockingCsMode(GlobalContext* globalCtx, Player* player) {
+    return (player->stateFlags1 & (PLAYER_STATE1_20000000 | PLAYER_STATE1_200 | PLAYER_STATE1_80)) ||
+           player->csMode != 0 || globalCtx->sceneLoadFlag == 0x14 || globalCtx->unk_18B4A != 0 ||
+           (player->stateFlags1 & PLAYER_STATE1_1) || (player->stateFlags3 & PLAYER_STATE3_80) ||
+           globalCtx->actorCtx.unk268 != 0;
+}
+
+s32 Player_InCsMode(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return Player_InBlockingCsMode(globalCtx, player) || player->unk_AA5 == 5;
+}
+
+s32 func_80123420(Player* player) {
+    return player->stateFlags3 & PLAYER_STATE3_80000000;
+}
+
+s32 func_80123434(Player* player) {
+    return player->stateFlags1 & (PLAYER_STATE1_40000000 | PLAYER_STATE1_20000 | PLAYER_STATE1_10000);
+}
+
+// Unused
+s32 func_80123448(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return (player->stateFlags1 & PLAYER_STATE1_400000) &&
+           (player->transformation != PLAYER_FORM_HUMAN || (!func_80123434(player) && player->unk_730 == 0));
+}
+
+s32 func_801234B0(Player* player) {
+    return player->transformation == PLAYER_FORM_GORON || player->transformation == PLAYER_FORM_DEKU;
+}
+
+s32 func_801234D4(GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    return (player->stateFlags2 & PLAYER_STATE2_8) || player->actor.speedXZ != 0.0f ||
+           ((player->transformation != PLAYER_FORM_ZORA) && (player->stateFlags1 & PLAYER_STATE1_8000000)) ||
+           ((player->transformation == PLAYER_FORM_ZORA) && (player->stateFlags1 & PLAYER_STATE1_8000000) &&
+            (!(player->actor.bgCheckFlags & 1) || (player->currentBoots < PLAYER_BOOTS_ZORA_UNDERWATER)));
+}
+
+s32 func_80123590(GlobalContext* globalCtx, Actor* actor) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    if ((player->stateFlags1 & PLAYER_STATE1_800) && (player->heldActor == actor)) {
+        player->interactRangeActor = NULL;
+        player->heldActor = NULL;
+        player->actor.child = NULL;
+        player->stateFlags1 &= ~PLAYER_STATE1_800;
+        return 1;
+    }
+    return 0;
+}
+
+s32 func_801235DC(GlobalContext* globalCtx, f32 arg1, s16 arg2) {
+    Player* player = GET_PLAYER(globalCtx);
+
+    if (player->stateFlags3 & PLAYER_STATE3_1000) {
+        player->unk_B08[0] = arg1;
+        player->unk_B08[1] += arg1 * 0.05f;
+        player->currentYaw = arg2;
+        player->actor.home.rot.y = arg2;
+        player->actor.shape.rot.y = arg2;
+        player->unk_B8C = 4;
+        player->invincibilityTimer = 20;
+
+        return true;
+    }
+
+    return false;
+}
```

Comment:
Are these actually used differently, then, that one returns a boolean and the other does not?

Reply:
No, not really. I just forgot to change that function to use true/false